jetpack-io/devbox
# vscode extension settings + devcontainer setup + devbox commands

## Summary
This doesn't fully implement the behaviour we discussed. But it does update the python interpreter and go path for the user.
The dev container setup is also possible via command palette (cmd + shift + P) in vscode.
- Added settings to the extension
- Added auto generating dev container config files
- cleaned up the code for more readability
- Added devbox commands (`devbox init | add | shell | install`) to the extension

## How was it tested?
- Open vscode-extensions/ in vscode
- press Run+debug
- check the prompt for update settings from devbox extension
- press cmd+shift+p and type devbox > choose "Devbox: Generate Dev Containers config files"
- check the generated files in .devcontainer/
